src/components/header/HeaderMenu.tsx

```diff
@@ -0,0 +1,113 @@
+import { useMemo } from "react";
+import { Link, useLocation } from "react-router-dom";
+import classNames from "classnames";
+import { Paths } from "../../routes/RoutesConfig";
+
+type Menu = {
+  dashboard: {
+    title: string;
+  };
+  reports: {
+    title: string;
+  };
+  manualUpload: {
+    title: string;
+  };
+  excelUpload: {
+    title: string;
+  };
+};
+
+const menuOptions: Menu = {
```

Comment:
**Optional thing to consider**: I'd extend this a bit and then I'd use it to build the menu based on this. For example:
_pseudo code_
```
manuOptions: {
    dashboard: {
        id: 'dashboard', // not required, but will be easier to access if needed
        title: 'Dashboard',
        path: '/dashboard'
    },
    reports: {
        id: 'reports',
        title: 'Reports',
        path: '/reports',
        subMenu: {
            raport1: { ... },
            raport2: { ... },
        },
    },
    ...
}
```
and then you could have a more generic HeaderMenu without a ton of duplicated code (in the future).

Reply:
@KarolinaKopacz and@sniedzielski what do you think about it?